Sources/CoffeeBot/CoffeeManager/CoffeeManager.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  CoffeeManager.swift
+//  SwiftBot
+//
+
+import Foundation
+import BotsKit
+import LoggerAPI
+
+internal protocol CoffeeManagerProtocol {
+    func process(activity:Activity)
+}
+
+class CoffeeManager {
+    
+    //MARK: Properties
+    fileprivate var session : Session? = nil
+    fileprivate let textProcessingService : TextProcessingService
+    fileprivate let appointingService : AppointingService
+    fileprivate let sessionService : SessionService
+    weak var botDelegate : CoffeeBot?
+    
+    //MARK: Lifecycle
+    init(textProcessingService:TextProcessingService, appointingService:AppointingService, sessionService:SessionService) {
```

Comment:
I mean create protocols for each service and then create implementation for each protocol, this can help you to tests your bot